Bottom: b8bd8de6044e86bb37b9827723a1b82a3fbbc83b
Top:    62b3622c228e49c40bdd3e05e9f9a8705b177352
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 11:42:52 +0000

Improve poly_AndSubtract_free() with similar tricks to poly_Boolean_free()





---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8da586a..f5bdc0f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2707,7 +2707,7 @@ poly_AndSubtract_free (POLYAREA * ai, POLYAREA * bi,
 #endif
       M_POLYAREA_intersect (&e, a, b, TRUE);
 
-      M_POLYAREA_label (a, b, FALSE);
+//      M_POLYAREA_label (a, b, FALSE);
       M_POLYAREA_label (b, a, FALSE);
 
 //      printf ("3:");
